app/src/main/res/layout/nightmode.xml

```diff
@@ -67,19 +75,19 @@
                             android:textColor="?attr/fontColor"/>
 
                     <RadioButton
-                            android:id="@+id/blue"
```

Comment:
On these changes, I just moved all the options in Night Mode around to match their order in the Base Theme menu. All that were there remain. Just added Pixel and Deep as options